nautobot/extras/api/serializers.py

```diff
@@ -643,6 +643,7 @@ class Meta:
             "description_override",
             "installed",
             "enabled",
+            "is_job_hook_receiver",
```

Comment:
Should perhaps be flagged as `read_only` in the API? We don't want this to be editable, right?

Reply:
That's picked up by the default serializer field since the model field is `editable=False`

```py
>>> js=JobSerializer()
>>> js.get_fields()["is_job_hook_receiver"]
BooleanField(help_text='Whether this job is a job hook receiver', read_only=True)
```

Reply:
😎